internal/k8s/pod.go

```diff
@@ -64,3 +64,66 @@ func imgPodMapFromOutput(output string) (map[string][]PodID, error) {
 	}
 	return imgsToPods, nil
 }
+
+func (k KubectlClient) GetNodeForPod(ctx context.Context, podID PodID) (NodeID, error) {
+	jsonPath := "-o=jsonpath={.spec.nodeName}"
+	b, err := k.kubectlRunner.cli(ctx, fmt.Sprintf("get pods %s '%s'", podID.String(), jsonPath))
+
+	out := b.String()
+
+	if err != nil {
```

Comment:
should this be handled by kubectlRunner?

Reply:
yep, thanks